docs/source/tooling/remix.rst

```diff
@@ -0,0 +1,64 @@
+.. _tools.remix:
+
+Remix
+=====
+
+MythX is available as a plugin for `Remix <https://remix.ethereum.org>`_, a popular web-based IDE for smart contract development and deployment, created and hosted by the Ethereum Foundation.
+
+.. image:: img/remix.jpg
+
+Setup
+-----
+
+.. note:: These instructions will show the Remix interface that is current as of mid-2019. We recommend everyone use this interface.
+
+Because Remix is a web-based interface, no local installation in required. However, MythX will need to be activated specifically before use.
+
+To activate MythX in Remix:
+
+#. Click the :guilabel:`Plugins` icon (which resembles a plug).
+
+   .. figure:: img/pluginsbutton.png
+
+   Alternately, click the :guilabel:`See all Plugins` button under :guilabel:`Featured Plugins`.
+
+#. The full list of plugins for Remix will be displayed. Scroll down to the entry titled :guilabel:`MythX Security Verification` and click :guilabel:`Activate`.
+
+   .. figure:: img/mythxpluginlist.png
+
+   If done correctly, the plugin will be listed under :guilabel:`Active Modules` and the MythX logo will be shown in the sidebar.
+
+   .. figure:: img/activemodules.png
+
+#. *(Optional but recommended)* Click the MythX logo and enter your MythX credentials. This consists of your Ethereum address (also known as User ID) and the password supplied to you when you created your account. When done, click :guilabel:`Save`.
```

Comment:
I think this should specify that the password is the one for MythX. "and the password supplied to you when you created your account in mythx.io"